File: guess.py
import random

guessesTaken = 0
number = random.randint(1, 20)

print('Hello, I am thinking of a number between 1 and 20.')

while guessesTaken < 6:
	print('Take a guess.')
	guess = input()
	guess = int(guess)
	guessesTaken = guessesTaken + 1

	if guess < number:
		print('Your guess is too low.')

	if guess > number:
		print('Your guess is too high.')

	if guess == number:
		number = str(number)
		guessesTaken = str(guessesTaken)
		print('You are a genius! You guessed my number in ' + guessesTaken + ' guesses!')

#print('Nope. The number I was thinking of was ' + number)

File: guess.js
var readlineSync = require('readline-sync');

const name = readlineSync.question('Hello! What is your name?\n');
console.log('Well, ' + name + ', I am thinking of  number between 1 and 20.')

const number = Math.floor(Math.random() * 20)

let guessesTaken = 0

while (guessesTaken++ < 6) {

  const guess = readlineSync.question('Take a guess\n');

  if (guess < number) {
    console.log('Too low!')
  } else if (guess > number) {
    console.log('Too high!')
  } else {
    console.log('Good job, ' + name + '! You guessed my number in ' + guessesTaken)
    return
  }

}

console.log('Nope. I was thinking of ' + number)